server.js

```diff
@@ -1897,6 +1883,55 @@ cache({
   }
 }));
 
+// npm type definition integration.
+camp.route(/^\/npm\/types\/(?:@([^/]+)\/)?([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache({
+  queryParams: ['registry_uri'],
+  handler: (queryParams, match, sendBadge, request) => {
+    // e.g. cycle, core, svg
+    const [, scope, packageName, format ] = match;
+    const apiUrl = makeNpmPackageDataUrl({
+      registryUrl: queryParams.registry_uri,
+      scope,
+      packageName,
+    });
+    const badgeData = getBadgeData('type definitions', queryParams);
+    request(apiUrl, { headers: { 'Accept': '*/*' } }, function(err, res, buffer) {
+      if (err != null) {
```

Comment:
Could we update this to use the [`checkErrorResponse`](https://github.com/badges/shields/blob/3cb71037fdbed20f74a18880ed9edad4273a08c5/lib/error-helper.js#L3-L19) helper:
https://github.com/badges/shields/blob/75503ec5144b349ab42c4877b774d3acac8157f3/server.js#L6515-L6518